core/metabolism.py

```
@@ -0,0 +1,266 @@
+        height: Height in cm
+        gender: "male" or "female"
+        body_fat: Optional body fat percentage (for Katch-McArdle formula)
+        formula: Formula to use ("mifflin", "harris", "katch", "cunningham")
+
+    Returns:
+        BMR in kcal/day
+
+    Raises:
+        ValueError: If unknown formula or missing body_fat for katch/cunningham
+    """
+    if age <= 0:
```

Ruff (0.13.3)

Avoid specifying long messages outside the exception class

(TRY003)